policyengine_canada/parameters/gov/provinces/ab/tax/income/credits/credits_return/lower_dependant_income_threshold.yaml

```diff
@@ -0,0 +1,13 @@
+description: Alberta limits the caregiver amount to dependants with income below this threshold.
+values:
+  2022-01-01: 17_826
+  2023-01-01: 19_331 
+metadata:
+  unit: currency-CAD
+  period: year
+  label: Alberta caregiver amount dependent lower income threshold
+  reference:
+    - title: Government of Canada - Alberta Caregiver Amount (2022)
```

Comment:
need to find legal code for all yaml files

Reply:
Hi Pavel, I'm having a lot of trouble finding some of the legal references. Could you help me out with that tomorrow? Except for that, I've fixed all necessary changes -- requesting a review.